Mondee Watch App/View/ArrowAnimation.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ArrowAnimation.swift
+//  Mondee Watch App
+//
+//  Created by 하명관 on 2023/07/13.
+//
+
+import SwiftUI
+
+struct ArrowAnimation: View {
+    @State var fade: Double = 0
+    @State var isAnimating: Bool = false
+    
+    var body: some View {
+        VStack(spacing: 2) {
+            ForEach(0..<3) { i in
```

Comment:
타입 추론 추가 및 chevronCount 변수 추가해보는건 어떤가요?
```suggestion
struct ArrowAnimation: View {

    @State var fade = 0.0
    @State var isAnimating = false
    
    private let chevronCount = 3
    
    var body: some View {
        VStack(spacing: 2) {
            ForEach(0..<chevronCount, id: \.self) { i in
```

Reply:
XCODE에 실행해보니 잘 되는군요!
가독성에 더 좋은 코드인것 같아요!